src/Ocelot/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -65,4 +69,22 @@ public static IOcelotBuilder AddOcelotUsingBuilder(this IServiceCollection servi
     {
         return new OcelotBuilder(services, configuration, customBuilder);
     }
+
+    private static IConfiguration MakeDefaultConfiguration(IWebHostEnvironment env)
+    {
+        return new ConfigurationBuilder().AddOcelot(env).Build();
+    }
```

Comment:
```suggestion
    private static IConfiguration MakeDefaultConfiguration(IWebHostEnvironment env)
        => new ConfigurationBuilder().AddOcelot(env).Build();
```

Reply:
@ArwynFr  Do you have time today to fix these 3 minor issues? 😉 
Or, if no objections then I will make one commit today.

Reply:
Sorry didn't see your message, please help yourself I won't be available until tomorow

Reply:
Committed as https://github.com/ThreeMammals/Ocelot/pull/1986/commits/715a3641920d02332cc22628ce3d9a3040b645dc